tests/loading_from_clipboard/loading_from_clipboard_test.cpp

```diff
@@ -114,12 +123,142 @@ namespace {
         DecklistBuilder decklistBuilder = DecklistBuilder();
         deckList->forEachCard(decklistBuilder);
 
-        CardRows expectedMainboard = CardRows({{"CardThatDoesNotExistInCardsXml", 1}});
+        CardRows expectedMainboard = CardRows({
+                                                      {"cardthatdoesnotexistincardsxml", 1}
```

Comment:
CLion did it and i couldn't change it no matter what I did... maybe you can help?